robot/EDA/resources/EDA.robot

```diff
@@ -175,6 +212,61 @@ Create Contact
     Store Session Record      Contact  ${contact_id}
     [return]                  ${contact_id}
     
+
+
+
+API Create Advising Or Walkin Appointment
+    [Documentation]
+    ...                     Creates an advising or walkin appointment through API. Returns dictionary representing the created advising appointment
+    ...
+    ...                     Required parameters are:
+    ...
+    ...                     |   location        |   value of Location field. Current existing values: 'In person' and 'By phone'    |
+    ...                     |   advising_type   |   value of Topic field. Ex: 'Academic'    |
+    ...                     |   advising_topic  |   value of Subtopic field. Ex: 'Degree Planning', 'Grade Concerns'    |
+    ...                     |   StartDate       |   Start time of the appointment. Format should be YYYY-MM-DD HH:MI:SS |
+    ...                     |   EndDate         |   End time of the appointment. Format should be YYYY-MM-DD HH:MI:SS   |
+    ...                     |                   |   Any additional field-value pairs can be passed too  |
+
+    [Arguments]             ${location}   ${advising_type}   ${advising_topic}    ${StartDate}    ${EndDate}    &{fields}
```

Comment:
Instead of having one super-long line, you might want to consider breaking this down into two lines for readability

```
[Arguments]  ${location}   ${advising_type}  ${advising_topic}
...          ${StartDate}  ${EndDate}        &{fields}
```

Reply:
@boakley - yeah, this code was inherited and included by someone else from some other work item.  I'm happy to include it to be removed/altered when I do the EDA rewrite.  But this is beyond the scope of this particular work item:  Preferred Phone.